plugins/org.jkiss.dbeaver.ext.mssql/src/org/jkiss/dbeaver/ext/mssql/model/SQLServerDataSource.java

```diff
@@ -93,7 +98,7 @@ public String getServerVersion() {
         return serverVersion;
     }
 
-    private boolean isNotBabelfishDatasource() {
+    public boolean isNotBabelfishDatasource() {
         boolean isBabelfish = SQLServerUtils.isDriverBabelfish(getContainer().getDriver());
         return !isBabelfish;
     }
```

Comment:
I don't quite follow the reason why this method is negated. Also, what is the point of having this method? It's just a wrapper that calls a utility function. Can we inline it?

Reply:
The reason for this method is that the expression has already shared between three different indirectly referenced methods implementing visibility conditions. (see [this commit](https://github.com/dbeaver/dbeaver/pull/14889/commits/e45306f3b42e33cb78eb1e413530ed30acfb028d))